.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+Thanks for contributing!
+-->
+
+Some things to make sure you've done:
+* Ran `swiftlint` and ensured that your additions/deletions are not causing any
+new warnings or errors.
+* You did _not_ include any changes to the generated documentation (we update
```

Comment:
I would suggest moving this to the bottom.

Reply:
Done.